ORM/DataObjectSchema.php

```diff
@@ -137,6 +138,62 @@ public function baseDataTable($class) {
 	}
 
 	/**
+	 * Get all DB field specifications for a class, including ancestors and composite fields.
+	 *
+	 * @param string|DataObject $classOrInstance
+	 * @param array $options Array of options. Specify any number of the below:
+	 *  - `uninherited`: Set to true to limit to only this table
+	 *  - `dbOnly`: Exclude virtual fields (such as composite fields), and only include fields with a db column.
+	 *  - `includeClass`: If true prefix the field specification with the class name in RecordClass.Column(spec) format.
+	 * @return array
+	 */
+	public function fieldSpecs($classOrInstance, $options = []) {
```

Comment:
Being really nitpicky but... could we use a bitmask instead of an array of case-sensitive strings for passing these options around? Personal preference, so feel free to say no if you disagree!